utils/mega_data_setup.py

```
@@ -689,10 +707,24 @@ def make_commodity_ah_api_request(self):
             time.sleep(1)
             raise Exception(error_message)
 
+        current_time = int(datetime.now().timestamp())
```

Ruff (0.8.2)

Local variable `current_time` is assigned to but never used

Remove assignment to unused variable `current_time`

(F841)